pywbem/_subscription_manager.py

```diff
@@ -463,13 +476,13 @@ def add_listener_destinations(self, server_id, listener_urls, owned=True):
 
         The form of the `Name` property of the created destination instance is:
 
-          ``"pywbemdestination:" {ownership} ":" {subscription_manager_id} ":"
-          {guid}``
+          ``"pywbemdestination:" {ownership} ":" {client_system_name} ":"
+          {subscription_manager_id} ":" {guid}``
 
         where ``{ownership}`` is ``"owned"`` or ``"permanent"`` dependent on
-        the `owned` argument; ``{subscription_manager_id}`` is the
-        subscription manager ID; and ``{guid}`` is a globally unique
-        identifier.
+        the `owned` argument; ``{client_system_name}`` is the fqdn of the client
+        system; ``{subscription_manager_id}`` is the subscription manager ID;
+        and ``{guid}`` is a globally unique identifier.
```

Comment:
In the description of the "subscription_manager_id" parameter, we talk about the "current host" in which the ID must be unique. I suggest that we use consistent terms, so change in the text above:

"{client_system_name}" to "{current_host}"

Reply:
Agree